client/main.js

```diff
@@ -16,15 +16,15 @@ import {
   updateInferenceText,
   error,
   setButtonsState,
-} from './ui.js';
+} from './components/ui.js';
 
 const state = {
   isCollectionOn: false,
   isInferenceOn: true,
   label: true,
   video: undefined,
   // TODO Consider putting the pathname to a JS[ON]/YAML file which would also be read by app.js
-  isInDevMode: window.location.pathname === '/dev',
+  isInDevMode: process.env.TRAIN_MODE,
```

Comment:
Would this still allow both pages to work as usual when being simultaneously built?

Reply:
Okay yep I've made a change that fixes this by passing both the html files as entry point. Only caveat of that is that we have to navigate to `/dev.html` with the explicit extension for it to work... root works fine

Reply:
Oh, I see! I think it's fine to have to type `/dev.html`, we could reduce it to `/dev` with some routing _magic_ or by renaming `dev.html` to `dev` so it's fine (especially if we end up adding a link/button from `index.html` to `dev.html`).